tools/scripts/acr-build.ps1

```diff
@@ -116,19 +128,24 @@ if ([string]::IsNullOrEmpty($sourceTag)) {
         $sourceTag = "latest"
     }
 }
-
-# Get build root - this is the top most folder with .dockerignore
```

Comment:
Is this safe to remove?

Reply:
I would say so as build and tests are OK.